domains/jake.json

```
@@ -1,11 +1,8 @@
 {
-  "description": "Jake's personal website and blog.",
-  "repo": "https://github.com/j-eo/j-eo.github.io",
   "owner": {
-    "username": "j-eo",
-    "email": "website-contact.nem2h@silomails.com"
+    "username": "xzrci"
   },
```

Email key is needed, even tho you dont want it.
Just do this
```suggestion
  "owner": {
  "email": "",
  "username: "xzrci"
  },
```

---

i have added, thanks